MonoGame.Framework/Input/MessageBox.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Xna.Framework.Input
+{
+    public partial class MessageBox
```

Comment:
@Nezz - Can we make this a static class?  Or is partial not allowed on static classes?

My problem is that in the docs...
 
  http://www.monogame.net/documentation/?page=T_Microsoft_Xna_Framework_Input_MessageBox

... it shows you can create an instance of `MessageBox` which is obviously not the intended use.

If not making it a static class... maybe make the default constructor private?